etc/testing/loads/load-1.yaml

```diff
@@ -10,7 +10,9 @@ operations:
               - source: "random"
                 prob: 100
         prob: 100
-validator: {}
+validator:
+  frequency:
+    prob: 0 
```

Comment:
So the validator is enabled, but it has zero probability of checking anything?

Reply:
Yeah, it needs to be enabled for deletions, but we don't want to run the validator.